src/Microsoft.PowerShell.Commands.Utility/commands/utility/ImplicitRemotingCommands.cs

```diff
@@ -2800,15 +2800,16 @@ private void GenerateHelperFunctions(TextWriter writer)
 
```

Comment:
@vexx32, your last commit had 22 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

JEA session Get-Help test.Get-Help should work in JEA sessions
```powershell
Exception calling "Invoke" with "0" argument(s): "Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed."
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Remoting\RemoteSession.Basic.Tests.ps1: line 139
139:             $helpContent = [powershell]::Create().AddScript($scriptBlock).Invoke()

```
Remoting loopback tests.Can execute command in a disconnected session
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
at <ScriptBlock>, D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1: line 226
226:     Invoke-Command @parameters

```
Remove-Item UnAuthorized Access.Access-denied test for removing a folder
```powershell
Expected $true, but got $false.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\FileSystem.Tests.ps1: line 1448
1448:         Wait-FileToBePresent -File $errorFile -TimeoutInSeconds 10 | Should -BeTrue
```
TestImplicitRemotingBatching hook should correctly batch simple remote command pipelines.Error occurred in Describe block
```powershell
Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed.
At D:\a\1\s\test\tools\Modules\HelpersRemoting\HelpersRemoting.psm1:191 char:20
```
JEA session Transcript script test.Configuration name should be in the transcript header
```powershell
Exception calling "Invoke" with "0" argument(s): "Objects cannot be added to a closed buffer. Make sure the buffer is open for Add and Insert operations to succeed."
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Remoting\RemoteSession.Basic.Tests.ps1: line 98
98:             [powershell]::Create().AddScript($scriptBlock).Invoke()

```